feat(changelog): transactional-email-added-webhooks-ga-release-track-and 2024-09-25 (#3749)
scaleway/docs-content: changelog/september2024/2024-09-25-transactional-email-added-webhooks-ga-release-track-and.mdx

---

> title: Webhooks GA Release: Track and React to Email Events Instantly

Reviewer: ```suggestion
title: Webhooks GA Release - Track and React to Email Events Instantly
```

> title: Webhooks GA Release - Track and React to Email Events Instantly

Reviewer 2: ```suggestion
title: Webhooks GA release - track and react to email events instantly
```

status: added
author:
    fullname: 'Join the #transactional-email channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-09-25
category: managed-services
product: transactional-email
---

Receive instant notifications on your email status (queued, delivered, dropped ...) and trigger automated actions in your integration. Simply set up your endpoint URLs and choose the events you want to track for quick and seamless reactions!

> Learn more (https://www.scaleway.com/en/docs/managed-services/transactional-email/how-to/create-webhooks/)

Reviewer: ```suggestion
Refer to the [How to create webhooks](https://www.scaleway.com/en/docs/managed-services/transactional-email/how-to/create-webhooks/) documentation page for more information.
```

> Refer to the [How to create webhooks](https://www.scaleway.com/en/docs/managed-services/transactional-email/how-to/create-webhooks/) documentation page for more information.

Reviewer 2: ```suggestion
Refer to the [How to create webhooks](/managed-services/transactional-email/how-to/create-webhooks/) documentation page for more information.
```
